frb_dart/bindgen/io_dartcobject.h

```diff
@@ -0,0 +1,124 @@
+#include <stdbool.h>
```

Comment:
Wait... I remember someone (you or maybe @rogurotus IIRC) already copies Dart SDK source code into our frb source code (and setup CI to ensure it is not stale). So maybe we should reuse that infrastructure to make it much easier

Reply:
Correct me if I'm wrong but I think we're not doing the same thing: this infrastructure copies dart sdk into `frb_rust` to include headers at user codegen time. My problem is to have `DartCObject` bindings at maintainer (us) coding time, in order to generate the dart object with correct static typing.

I'd love to reuse such logic but it probably can't be done as-is. However if you meant to use original dart sdk rather than generating bindings from `allo_isolate` well I sure can look into that.

Adding another `cp` in `code_generator.yml:66` to copy those headers in `frb_dart` as well sound fine to you?